go.mod

```diff
@@ -185,11 +192,6 @@ replace (
 	// TODO - Remove this patch and update Tokens in a subsequent upgrade handler
 	github.com/cosmos/cosmos-sdk => github.com/Stride-Labs/cosmos-sdk v0.47.4-stride-distribution-fix-1
 
-	// Two changes
-	// (1) Testing infra
-	// (2) Fix bech32 bug
-	github.com/cosmos/interchain-security/v3 => github.com/Stride-Labs/interchain-security/v3 v3.1.0-remove-validation-bug-7d3d9d
```

Comment:
Amazing to get off the fork! 

Could you pls add a short comment to the PR description describing why we were on a fork before and why we can come off it now, just for future reference?

Reply:
added!